### DISCUSSION TITLE
Operations on superslicer variables

### DISCUSSION THREAD

**Discussion by GrimSinner** [supermerill/SuperSlicer#2013] [Q&A]
What is the syntax for adding +5 to a variable in SS?  For example, I want to add 5C to the initial bed temperature in my start gcode.  If my first layer is 65C I want the start code to use 70C, like M140 S[first_layer_bed_temperature]+5.  When I look at the gcode, it shows M140 S65+5, instead of M140 S70.

**Comment by supermerill** [comment]
S{first_layer_bed_temperature[current_extruder]+5}

'[]' are a shortcut, kind of old syntax. the '{}' is where you can really script things. 

https://help.prusa3d.com/en/article/macros_1775
